### PR TITLE
list apphooks cleanup

### LINKED ISSUE #3776: `list apphooks` should sort and clarify its output
Currently if you run `./manage.py cms list apphooks` you'll get slightly confusing output:

    VideosApp
    EventApp
    VideosApp
    EventApp
    PressReleaseApp
    NewsApp
    PressReleaseApp
    NewsApp

This could be consolidated to something like:

    EventApp (draft/published)
    NewsApp (draft/published)
    PressReleaseApp (draft/published)
    VideosApp (draft/published)

If the draft/published state is unnecessary, then it can be omitted.

### PR DESCRIPTION
**Pull request by yakky**
Fix #3776 

Proposed output:

    Example1App (draft)
    NamespacedApp[instance: namespaced_app_ns] (draft)
    ChildApp (draft)

### CODE REVIEW

**Comment by yakky** [comment]
@FinalAngel @mkoistinen Please review

File: cms/tests/test_management.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def test_list_apphooks(self):
         command = cms.Command()
         command.stdout = out
         command.handle("list", "apphooks", interactive=False)
-        self.assertEqual(out.getvalue(), "SampleApp\n")
+        self.assertEqual(out.getvalue(), "SampleApp (draft)\n")
```
Comment (FinalAngel) [inline cms/tests/test_management.py]: single quotes?

Comment (yakky) [inline cms/tests/test_management.py]: single/double quotes are still a bit messy in the CMS. I prefer single to double but the module is almost all double and I think it's better to do a global cleanup in 3.3

**Comment by yakky** [comment]
No docs needed as it's just a display change

**Comment by FinalAngel** [comment]
alright LGTM :)

**Comment by yakky** [comment]
@intellectronica , @mkoistinen any comment?

**Comment by mkoistinen** [comment]
Looks real good! :+1:

**Pull request merged**